app/views/maps/_page_map_infowindow.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="line-height-sans-1">
+  <div class="margin-top-1">
+    <%
+      station_name = facility.official_station_name
+      facility_id = facility.id
+      practice_list = @practices_list
+      names = []
+      taglines = []
+      short_names = []
+      slugs = []
```

Comment:
With the recommendation above, we would not need all of these array variables

Reply:
Done